ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -178,21 +178,21 @@ open class SearchBar: UIView {
 
     weak var navigationController: NavigationController?
 
-    //used to hide the cancelButton in non-active states
+    /// used to hide the cancelButton in non-active states
```

Comment:
for private function likes these // is the right format. /// is used for public classes that you are showing documentation when developer right clicks on the class to find out more info.

Reply:
fixed in 3rd iteration

---
In reply to: [467169551](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r467169551) [](ancestors = 467169551)